### ISSUE TITLE
usr/bin/ld: /usr/local/lib/libAKAZE.so: undefined reference to `cudaMemcpy2DAsync

### ISSUE THREAD

**Issue by harendracmaps** [pablofdezalc/akaze#38]
Hi ,

I am trying to compile the library on x86_64.  I have the following graphics card. 
```
$ lspci | grep VG
01:00.0 VGA compatible controller: NVIDIA Corporation GP104M [GeForce GTX 1070 Mobile] (rev a1)
```
I have installed cuda-toolkit 10.0 version. I can compile it successfully and generate libAKAZE_CUDA.so alibAKAZE.so. But when I try to link it with my program I am getting the follwoing error 
```
 /usr/local/lib/libAKAZE.so: undefined reference to `cudaMemcpy2DAsync
```
I tried linking it with **-lAKAZE -lAKAZE_CUDA** options.

If I run the ldd command I am not getting any output. 
```
$ ldd lib/libAKAZE_CUDA.so | grep cuda
```

but it I try compiling it on nvidia Tx2  it is able to link successfully and If I run the ldd command on tx2 I am getting output. 
```
$ ldd lib/libAKAZE_CUDA.so | grep cuda
libcudart.so.10.0 => /usr/local/cuda-10.0/lib64/libcudart.so.10.0 (0x0000007f8ef1a000)
```

Please let me know if any more info is needed. Appreciate if someone please help me overcome this issue.

**Comment by harendracmaps** [comment]
fixed by renaming the /usr/local/cuda-10.0/lib64/libcudart_staic.a library to /usr/local/cuda-10.0/lib64/libcudart_staic.a.bkp

**Comment by fehimebetul** [comment]
Hi Sir,
can you tell us how it runs on tx2?
